lib/doorkeeper/config/validations.rb

```diff
@@ -16,6 +17,16 @@ def validate!
 
       private
 
+      def validate_client_authentication_value
+        return if client_authentication.is_a?(Array)
+
+        ::Rails.logger.warn(
```

Comment:
I can't figure out a way to test this, but this is where supporting a list of arguments for `option` would be a good idea.